app/navigation/ManufacturingNavigator.scala

```diff
@@ -28,14 +28,14 @@ class ManufacturingNavigator @Inject()() extends Navigator {
 
   protected val routeMap: PartialFunction[Page, UserAnswers => Call] = {
     case TypeOfManufacturingPage => typeOfManufacturing(NormalMode)
+    case AluminiumApprenticePage => aluminiumApprentice(NormalMode)
     case AluminiumOccupationList1Page => aluminiumOccupationList1(NormalMode)
     case AluminiumOccupationList2Page => aluminiumOccupationList2(NormalMode)
-    case AluminiumOccupationList3Page => aluminiumOccupationList3(NormalMode)
-    case AluminiumApprenticePage => _ => controllers.routes.EmployerContributionController.onPageLoad(NormalMode)
-    case IronApprenticePage => _ => controllers.routes.EmployerContributionController.onPageLoad(NormalMode)
+    case AluminiumOccupationList3Page => _ => controllers.routes.EmployerContributionController.onPageLoad(NormalMode)
+    case IronApprenticePage => _ => routes.IronMiningController.onPageLoad(NormalMode)
```

Comment:
i think if apprentice true it should go to employer contribution, then iron mining if false?